src/lib/navigationFilter.ts

```diff
@@ -26,7 +26,7 @@ const openBrowser = async (url: string) => {
 export const shouldLoadFilter = (request: ShouldStartLoadRequest) => {
   // Navigation within wrapped web app
   if (
-    request.url.startsWith(SHAPESHIFT_URI) ||
+    request.url.startsWith("https://app.shapeshift.com") ||
```

Comment:
Any reason why we don't keep this programmatic from the env var @NeOMakinG?

Reply:
No reason, I'm just dumb, I think I took too much time debugging the android icon and I forgot a few tests